cocos/spine/lib/spine-define.ts

```diff
@@ -732,7 +790,36 @@ function overrideProperty_RegionAttachment (): void {
     });
 
     overrideDefineArrayProp(prototype, prototype.getOffset, 'offset');
-    overrideDefineArrayPropGetSet(prototype, prototype.getUVs, prototype.setUVs, spine.wasmUtil.wasm.VectorFloat, 'uvs');
+    const getUVs = prototype.getUVs;
+    const setUVs = prototype.setUVs;
+    const _uvs = '_uvs';
+    Object.defineProperty(prototype, 'uvs', {
+        get (): any {
+            const vectors = getUVs.call(this);
+            const count = vectors.size();
+            let array = prototype[_uvs];
+            array = resizeArray(array, count);
+            for (let i = 0; i < count; i++) array[i] = vectors.get(i);
+            prototype[_uvs] = array;
+            return array;
+        },
+        set (value: number[]) {
+            setUVs.call(this, value[0], value[1], value[2], value[3], value[4] === 1);
+        },
+    });
+
+    const originComputeWorldVertices = prototype.computeWorldVertices;
+    Object.defineProperty(prototype, 'computeWorldVertices', {
+        value (bone: spine.Bone, worldVertices: number[], offset: number, stride: number) {
```

Comment:
Same question as above.

Reply:
const vectors = new spine.SPVectorFloat();
    Object.defineProperty(prototype, 'computeWorldVertices', {
        value (slot: spine.Slot, start: number, count: number, worldVertices: number[], offset: number, stride: number) {
            const length = worldVertices.length;
            vectors.resize(length, 0);
            for (let i = 0; i < length; i++) vectors.set(i, worldVertices[i]);
            originComputeWorldVertices.call(this, slot, start, count, vectors, offset, stride);
            for (let i = 0; i < length; i++) worldVertices[i] = vectors.get(i);
        },
    }); Optimize computeWorldVertices to avoid every call to new SPVectorFloat();.